src/backend/web/routes/query.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+
+const { logger } = require('../../utils/logger');
+const { search } = require('../../utils/elastic');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  // Github rule, query is more than 256 chars return a validation failed message
+  if (req.query.q.length > 256) {
```

Comment:
Can we change this so the parameter tells us something about what we're doing? Instead of `q` it could be `search` or something a bit clearer.

Reply:
Also, this a silly nit, but let's avoid [magic numbers](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544).